guide/interactions/modals.md

```diff
@@ -101,9 +101,11 @@ Restart your bot and invoke the `/ping` command again. You should see a popup fo
 
 Modals are received via an interaction. You can check if a given interaction is a modal by invoking the `#isModalSubmit()` method:
 
-```js {2}
+```js {1,4}
+const { InteractionType } = require('discord.js');
+
 client.on('interactionCreate', interaction => {
-	if (!interaction.isModalSubmit()) return;
+	if (!interaction.type === InteractionType.ModalSubmit) return;
```

Comment:
```suggestion
	if (interaction.type !== InteractionType.ModalSubmit) return;
```
`!interaction.type` is coerced to a boolean which will never be equal to an enum value